tools/walletextension/api/staticOG/javascript.js

```diff
@@ -6,10 +6,10 @@ const idAddAllAccounts = "addAllAccounts";
 const idRevokeUserID = "revokeUserID";
 const idStatus = "status";
 const idUserID = "userID";
-const pathJoin = "/join/";
-const pathAuthenticate = "/authenticate/";
-const pathQuery = "/query/";
-const pathRevoke = "/revoke/";
+const pathJoin = "/v1/join/";
```

Comment:
Maybe good to have API version constant like in the Go to build all these URLs with?

Reply:
Added, thanks!